src/libcollections/string.rs

```diff
@@ -1382,6 +1382,71 @@ impl String {
         }
     }
 
+    /// Creates a splicing iterator that removes the specified range in the string,
+    /// replaces with the given string, and yields the removed chars.
+    /// The given string doesn’t need to be the same length as the range.
+    ///
+    /// Note: The element range is removed even if the iterator is not
+    /// consumed until the end.
```

Comment:
How about

    /// Note: The element range is removed when the Splice is dropped
    /// even if the iterator is not consumed until the end.